### PR TITLE
Add querystring values

### PR DESCRIPTION
**Pull request by hatchan**
Adds querystring values for "list user application", "list builds by application", "list deploys by application".

### CODE REVIEW

File: content/api/endpoints/applications.md [hunk 1]
```diff
@@ -98,6 +107,19 @@ Retrieve all builds of an application.
...
+
+| Name | Description |
+|------|-------------|
+| `branch` | Branch of the commit that triggered a build. |
+| `commit` | Commit of the build that triggered the build. `40` characters. |
+| `result` | Result of the build. Possible values: `aborted`, `unknown`, `passed`, `failed`. |
+| `stack` | Stack used to run a build. Currently supported: `1` (classic), `5` (Docker enabled) |
+| `status` | Status of the build. Possible values: `notstarted`, `started`, `finished`, `running`. |
+| `limit` | Limit the results that will get returned. Default: `10`. Min: `1`. Max: `20`. |
+| `skip` | Skip a certain ammount of builds. Default: `0`. Min: `0`. |
```
Comment (yoshuawuyts) [inline content/api/endpoints/applications.md]: I suggest we omit `min` when `n=0`; I found it confusing when reading. We also don't write `max=infinity` when there's no max.

**Comment by yoshuawuyts** [comment]
I suggest we omit `min` when `n=0`; I found it confusing when reading. We also don't write `max=infinity` when there's no max.

Apart from that lgtm!

File: content/api/endpoints/applications.md (hunk 1 above)
Comment (hatchan) [inline content/api/endpoints/applications.md]: Guess that makes sense, changing

**Comment by hatchan** [comment]
This can be merged once wercker/wercker#906 get merged

**Pull request merged**